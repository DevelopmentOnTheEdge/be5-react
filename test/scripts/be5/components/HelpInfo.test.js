import React          from 'react';
import renderer       from 'react-test-renderer';
import HelpInfo       from '../../../../src/scripts/be5/components/HelpInfo';
import {TestProvider}      from "../testUtils";
import '../../../../src/scripts/be5/actions/text'


test('snapshot', () => {
  const component = renderer.create(
    <TestProvider>
      <HelpInfo value="text/Test text"/>
    </TestProvider>
  );

  expect(component.toJSON()).toMatchSnapshot();
});
