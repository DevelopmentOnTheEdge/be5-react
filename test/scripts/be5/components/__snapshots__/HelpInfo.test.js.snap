// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<div
  className="helpInfo clearfix"
>
  <button
    className="btn-sm btn btn-info"
    onClick={[Function]}
    type="button"
  >
    Help
  </button>
  <div
    className="collapse"
    style={Object {}}
  >
    <div
      className="alert alert-success max-width-970"
      role="alert"
    >
      <div
        className="document-content"
        id="document-content___helpInfo"
      >
        <div>
          <div
            className="staticPage"
          >
            <div
              className="staticPage__text"
              dangerouslySetInnerHTML={
                Object {
                  "__html": "Test text",
                }
              }
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
